### PR TITLE
Manage case of ingest with base directory root (/).

### PR DESCRIPTION
**Pull request by siscia**
It seems to me that this case was not manage yet.

### CODE REVIEW

**Comment by siscia** [comment]
@cernvm-bot cloudtest

**Comment by siscia** [comment]
Do not merge yet, it still need tests to cover the new feature.

File: cvmfs/sync_union_tarball.cc [hunk 1]
```diff
@@ -146,10 +146,12 @@ void SyncUnionTarball::Traverse() {
 
       case ARCHIVE_EOF: {
         if (create_catalog_on_root_) {
```
Comment (jblomer) [inline cvmfs/sync_union_tarball.cc:148]: Perhaps better `if (create_catalog_on_root_ && (base_directory_ != "/"))`

**Comment by siscia** [comment]
I added a simple test and, doing so I decided to slide other integration tests.

I tried to keep related tests all together, close to each other. I am not sure I succeeded.

File: cvmfs/sync_union_tarball.cc (hunk 1 above)
Comment (siscia) [reply cvmfs/sync_union_tarball.cc]: I agree!

**Comment by siscia** [comment]
@cernvm-bot cloudtest

**Comment by siscia** [comment]
@cernvm-bot cloudtest

**Comment by siscia** [comment]
@jblomer I believe we can merge this. The failures seems unrelated.

**Pull request merged**